cirq-google/cirq_google/engine/stream_manager.py

```diff
@@ -121,6 +121,11 @@ def __init__(self, grpc_client: quantum.QuantumEngineServiceAsyncClient):
         # interface.
         self._response_demux = ResponseDemux()
         self._next_available_message_id = 0
+        self._executor.submit(self._init_request_queue).result()
```

Comment:
It looks like `self._request_queue` is only ever accessed on the duet side (in `StreamManager.submit`). Is this to ensure that it gets bound to the correct event loop when constructed? If so, I would suggest just submitting a job to construct the queue but then assigning it here:
```python
        # Construct queue in executor to ensure it binds to the correct event loop
        self._request_queue  = self._executor.submit(self._make_request_queue).result()

    async def _make_request_queue(self) -> asyncio.Queue[QuantumRunStreamRequest]:
        return asyncio.Queue()
```
But would be good to check that this is needed. In the current implementation `asyncio.Queue` doesn't bind its event loop until either `get` or `put` are called so would be fine to construct it on the duet side and pass to the executor to use, but this may have changed in python 3.10 when the `loop` arg was removed.